certified-connectors/SIGNL4 - Mobile Alerting/apiProperties.json

```diff
@@ -0,0 +1,22 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key",
+          "description": "The API Key for this api",
+          "tooltip": "Provide your API Key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#000000",
+    "capabilities": [],
+    "publisher": "Ronald Czachara"
```

Comment:
@rons4 Please add the stackOwner property. Please see out template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiProperties.json). 
```
    "stackOwner": "Name of the owner of the service or product."
```

Reply:
Done.